internal/pkg/cli/svc_logs_test.go

```diff
@@ -106,6 +108,14 @@ func TestSvcLogs_Validate(t *testing.T) {
 
 			wantedError: fmt.Errorf("--limit 10001 is out-of-bounds, value must be between 1 and 10000"),
 		},
+		"returns error if both previous and tasks flags are defined": {
```

Comment:
Do we also have any unit tests for success with taskID(s)?

Reply:
Yes, I had implemented that unit test as a part of original ask of --previous flag. You can check the unit test [here](https://github.com/aws/copilot-cli/blob/e1ae307bda11ab8d8e940f9fefd398ced094bb16/internal/pkg/cli/svc_logs_test.go#L374)